ActyvityLog.js

```diff
@@ -59,14 +59,33 @@ _renderPts(data){
 }
 
 _renderTime(data){
-  if (data == null){
-    return 0
-  }else{
-    return data
+  if(data != null){
+    if (data.sec <= 9 && data.min <= 9){
+      return(
+        <Text style = {stylesActyvityLog.textData}>0{data.min}:0{data.sec}</Text>
```

Comment:
looks like var formattedNumber = ("0" +data.min).slice(-2) should work to insert "0" in the beginning if needed